config/prompts_memory.py

```
@@ -802,6 +802,273 @@ def get_fact_extraction_prompt(lang: str = 'zh') -> str:
+def get_negative_preference_review_prompt(lang: str = 'zh') -> str:
+    lang = (lang or 'zh').strip().split('-', 1)[0].split('_', 1)[0].lower() or 'zh'
+    return _loc(NEGATIVE_PREFERENCE_REVIEW_PROMPT, lang)
+
+
+negative_preference_review_prompt = NEGATIVE_PREFERENCE_REVIEW_PROMPT['zh']
+
+
+NEGATIVE_TOPIC_VALIDATION_PROMPT = {
+    'zh': """请判断下面这个“负面偏好 topic 候选”是否是一个正常、具体、可写入长期记忆的话题。
+
+目标：
```

Ruff (0.15.9)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)